Documentation/advanced-configuration.md

```diff
@@ -26,6 +27,78 @@ The Kubernetes based examples assume Rook OSD pods are in the `rook-ceph` namesp
 If you run them in a different namespace, modify `kubectl -n rook-ceph [...]` to fit
 your situation.
 
+## Use custom Ceph user and secret for mounting
+
+> **NOTE** For extensive info about creating Ceph users, consult the Ceph documentation: http://docs.ceph.com/docs/mimic/rados/operations/user-management/#add-a-user.
+> Using a custom Ceph user and secret can be done for filesystem and block storage.
+
+Create a custom user in Ceph with access to just read write access in `/bar` directory on CephFS:
+```
+ceph auth get-or-create-key client.user1 mon 'allow r' osd 'allow rw tag cephfs data=YOUR_FS_DATA_POOL' mds 'allow r, allow rw path=/bar'
+```
+The command will return a Ceph secret key, this key should be added as a secret in Kubernetes like this:
+```
+kubectl create secret generic ceph-user1-secret --from-literal=key=YOUR_CEPH_KEY
+```
+> **NOTE** This secret with the same name must be created in each namespace where the StorageClass will be used.
+> In addition to this Secret you must create a RoleBinding to allow the Rook Ceph agent to get the secret from each namespace.
+> The RoleBinding is optional if you are using a ClusterRoleBinding for the Rook Ceph agent secret access.
+> The Bindings are shown as an example after the next step.
+
+On a StorageClass `parameters` and/or flexvolume Volume entry `options` set the following options:
+```
+mountUser: user1
+mountSecret: ceph-user1-secret
+```
+
+### RoleBinding
+
+Create it in each namespace the Rook Ceph agent should read secrets for mounting.
```

Comment:
Even though you said it in the previous section, I would suggest another statement in this section (and ClusterRoleBinding section) again that you need **one or the other**